### PR TITLE
feat: add Ramp

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-10794](https://app.clickup.com/t/2645698/DEV-10794)

### Description

- integrate [Ramp](https://ramp.network/) to buy LYX
- optimize image sizes

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-10794 Add "Ramp" as an option to buy LYX](https://app.clickup.com/t/2645698/DEV-10794)

File: public/images/ramp-logo.png
Comment (doubleppereira) [inline public/images/ramp-logo.png:1]: The quality of this one looks a bit blurry no?

**Review by doubleppereira — APPROVED** [review]
LGTM just one image looks a bit blurry

File: public/images/ramp-logo.png
Comment (dzbo) [reply public/images/ramp-logo.png:1]: It's ok I've tested in Retina screen, we use it as small image.

**Pull request merged**